### ISSUE TITLE
Increase reliability of adding/moving multiple playlist songs

### ISSUE THREAD

**Issue by thebigmunch** [thebigmunch/google-music#3]
With some workarounds and checks currently implemented, it is more reliable than it would be otherwise. But, it's still not reliable. Adding/moving too many in a short time can cause the server to get behind in syncing the mutations. It could be possible to add some more/different workarounds and checks to help with this in the future.

**Comment by thebigmunch** [comment]
For the record, there is a ``plentries`` endpoint that may be an alternative to ``plentriesbatch`` in the same way that ``playlists`` is an alternative to ``playlistbatch``. Not sure if it will provide any improvement, but I'll figure the endpoint out to test with at some point.

**Comment by thebigmunch** [comment]
Alright, I believe I got this one licked : )

**Comment by thebigmunch** [comment]
I'm bringing the implementations back to using just one batch call. While testing things out, I noticed that despite the earlier bugs I noticed in Google's client still happening, I am now getting consistent, proper results making just one call. There may be a difference with using ``'id'`` vs ``'clientId'``. It could be a fluke. Also, the previous workaround made the methods take a prohibitively long time to finish for even relatively small numbers of songs. So either way, I believe it's best just to deal with what Google gives us.
